src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -640,4 +637,29 @@ private Article entityToArticle(Entity entity){
     }
     return article;
   }
+
+  public void updateFieldForum(String forumName, String fieldToAppend, String valueToAppend, boolean updateAll){
+    PreparedQuery results = null;
+    if(updateAll){
+      Query query =
+              new Query("Forum")
+              .addSort("title",SortDirection.ASCENDING);
```

Comment:
Do we need `addSort` since we are updating all forums?

Reply:
it was used for debugging purpose.